README.md

```diff
@@ -254,6 +254,16 @@ val notA = !a // or a.isFalse()
 
 Furthermore onTrue() and onFalse() are defined and will give a ReadChannel[Unit] that trigger when either true or false value is received.
 
+Aritmetic operations like +, -, *, /, %, <, <=, >, >=, ===, !=== are also supported for types that define them (supported by Numeric/Ordering traits):
```

Comment:
Ah ok, maybe "!==" is better yes. I see that Slick is using "=!=", either = in front or behind... Pushed.